olp-cpp-sdk-dataservice-write/CMakeLists.txt

```diff
@@ -21,6 +21,16 @@ set(DESCRIPTION "C++ API library for writing data to OLP")
 file(GLOB_RECURSE DATASERVICE_WRITE_INC "include/*.h*")
 file(GLOB_RECURSE DATASERVICE_WRITE_SRC "src/*.*")
 
+
+list(REMOVE_ITEM DATASERVICE_WRITE_SRC "${CMAKE_CURRENT_SOURCE_DIR}/src/AutoFlushController.cpp"
+                                       "${CMAKE_CURRENT_SOURCE_DIR}/src/AutoFlushController.h"
+                                       "${CMAKE_CURRENT_SOURCE_DIR}/src/AutoFlushSettings.h"
+                                       "${CMAKE_CURRENT_SOURCE_DIR}/src/BackgroundTaskCollection.cpp"
+                                       "${CMAKE_CURRENT_SOURCE_DIR}/src/BackgroundTaskCollection.h"
+                                       "${CMAKE_CURRENT_SOURCE_DIR}/src/DefaultFlushEventListener.cpp"
+                                       "${CMAKE_CURRENT_SOURCE_DIR}/src/DefaultFlushEventListener.h"
+                                       "${CMAKE_CURRENT_SOURCE_DIR}/src/FlushEventListener.h")
+
```

Comment:
Let's rewrite the script to include only necessary files, instead of GLOB + REMOVE_ITEM